.pre-commit-config.yaml

```diff
@@ -5,3 +5,9 @@ repos:
     rev: 20.8b0
     hooks:
     - id: black
+      files: \.py$
+-   repo: https://github.com/pycqa/isort
+    rev: 5.6.4
+    hooks:
+      - id: isort
```

Comment:
The `isort` pre-commit hook modified some examples for me locally, looks like the CI only checks the `nengo_fpga` dir so it doesn't see the examples in `docs`. Should we update the examples even though they aren't checked by CI?

Perhaps a bigger `bones` question is if we want to change either pre-commit or CI so they match (same thing with `black`)?

Reply:
I ran `isort` on the `.py` files in the docs folder. For the notebooks, we'll wait until a similar functionality is available with the `bones-format-notebook` script.